crates/bitwarden-core/src/client/encryption_settings.rs

```diff
@@ -30,10 +30,20 @@ pub enum EncryptionSettingsError {
     MissingPrivateKey,
 }
 
+/// A struct containing the core keys of a Bitwarden user.
```

Comment:
**suggestion**: It's fairly self explanatory that this is a struct both from the code, but also in rust docs.

```suggestion
/// [EncryptionSettings] contains the encryption state related to a Bitwarden account.
```